apps/dashboard/src/components/app-layouts/provider-setup.tsx

```diff
@@ -103,52 +64,60 @@ export const DashboardThirdwebProviderSetup: ComponentWithChildren<
     };
   }, [chain]);
 
+  const ethersSigner = useEthersSigner();
+
   return (
-    <ThirdwebProvider>
-      <ThirdwebProviderOld
-        queryClient={queryClient}
-        dAppMeta={{
-          name: "thirdweb",
-          logoUrl: "https://thirdweb.com/favicon.ico",
-          isDarkMode: false,
-          url: "https://thirdweb.com",
-        }}
-        activeChain={activeChain}
-        supportedChains={supportedChains}
-        sdkOptions={{
-          gasSettings: { maxPriceInGwei: 650 },
-          readonlySettings,
-        }}
-        clientId={DASHBOARD_THIRDWEB_CLIENT_ID}
-        secretKey={DASHBOARD_THIRDWEB_SECRET_KEY}
-        supportedWallets={dashboardSupportedWallets}
-        storageInterface={StorageSingleton}
-        authConfig={{
-          domain: THIRDWEB_DOMAIN,
-          authUrl: `${THIRDWEB_API_HOST}/v1/auth`,
-        }}
-      >
-        <GlobalAuthTokenProvider />
-        {children}
-      </ThirdwebProviderOld>
-    </ThirdwebProvider>
+    <ThirdwebSDKProvider
+      queryClient={queryClient}
+      signer={ethersSigner}
+      activeChain={activeChain}
+      supportedChains={supportedChains}
+      sdkOptions={{
+        gasSettings: { maxPriceInGwei: 650 },
+        readonlySettings,
+      }}
+      clientId={DASHBOARD_THIRDWEB_CLIENT_ID}
+      secretKey={DASHBOARD_THIRDWEB_SECRET_KEY}
+      storageInterface={StorageSingleton}
+    >
+      {children}
+    </ThirdwebSDKProvider>
   );
 };
 
-const GlobalAuthTokenProvider = () => {
-  const { token, isLoading } = useApiAuthToken();
+function useEthersSigner() {
```

Comment:
wonder if we should just have this as a 'adapter hook' exported out from the sdk

Reply:
eh... maybe? Can add it as a follow-up